### DISCUSSION TITLE
how do i sponsor my project?

### DISCUSSION THREAD

**Discussion by bor-ges** [community/community#134667] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

can yout guys help?

**Comment by bor-ges** [comment]
sponsoring

**Comment by ebndev** [comment, marked as answer]
Please see [this comment](https://github.com/orgs/community/discussions/134669#discussioncomment-10246676) for why this discussion was closed and locked
